docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,234 @@
+
+```
+$ gcloud dns managed-zones describe example-dot-com 
+creationTime: '2016-06-26T18:18:39.229Z'
+description: Example domain for Kubernetes Cluster Federation
+dnsName: example.com.
+id: '3229332181334243121'
+kind: dns#managedZone
+name: example-dot-com
+nameServers:
+- ns-cloud-a1.googledomains.com.
+- ns-cloud-a2.googledomains.com.
+- ns-cloud-a3.googledomains.com.
+- ns-cloud-a4.googledomains.com.
+```
+
+```
+$ gcloud dns record-sets list --zone example-dot-com
+NAME                                                                                                 TYPE      TTL     DATA
+example.com.                                                                                       NS     21600  ns-cloud-e1.googledomains.com., ns-cloud-e2.googledomains.com.
+example.com.                                                                                      SOA     21600 ns-cloud-e1.googledomains.com. cloud-dns-hostmaster.google.com. 1 21600 3600 1209600 300
+nginx.mynamespace.myfederation.svc.example.com.                            A     180     104.197.246.190, 130.211.57.243, 104.196.14.231, 104.199.136.89,...
+nginx.mynamespace.myfederation.svc.us-central1-a.example.com.     A     180     104.197.247.191
+nginx.mynamespace.myfederation.svc.us-central1-b.example.com.     A     180     104.197.244.180
+nginx.mynamespace.myfederation.svc.us-central1-c.example.com.     A     180     104.197.245.170
+nginx.mynamespace.myfederation.svc.us-central1-f.example.com. CNAME 180     nginx.mynamespace.myfederation.svc.us-central1.example.com.
+nginx.mynamespace.myfederation.svc.us-central1.example.com.         A    180     104.197.247.191, 104.197.244.180, 104.197.245.170
+nginx.mynamespace.myfederation.svc.asia-east1-a.example.com.       A    180     130.211.57.243
+nginx.mynamespace.myfederation.svc.asia-east1-b.example.com.  CNAME 180     nginx.mynamespace.myfederation.svc.asia-east1.example.com.
+nginx.mynamespace.myfederation.svc.asia-east1-c.example.com.       A    180     130.211.56.221
+nginx.mynamespace.myfederation.svc.asia-east1.example.com.           A    180     130.211.57.243, 130.211.56.221
+nginx.mynamespace.myfederation.svc.europe-west1.example.com.  CNAME    180   nginx.mynamespace.myfederation.svc.example.com.
+nginx.mynamespace.myfederation.svc.europe-west1-d.example.com.  CNAME   180     nginx.mynamespace.myfederation.svc.europe-west1.example.com. 
+... etc.
+```
+
+Note: If your Federation is configured to use AWS Route53, you can use one of the equivalent AWS tools, for example:
+
+```
+$aws route53 list-hosted-zones
+```
+and
+```
+$aws route53 list-resource-record-sets --hosted-zone-id Z3ECL0L9QLOVBX
+```
+
+Whatever DNS provider you use, any DNS query tool (for example 'dig' or 'nslookup') will of course also allow you to see the records created by the Federation for you.
```

Comment:
Actually not, if you point them straight at your DNS provider, but yes, worth adding a note to prevent people running foul of TTL's